configure

```diff
@@ -288,6 +288,11 @@ parser.add_option('--enable-d8',
     dest='enable_d8',
     help=optparse.SUPPRESS_HELP)  # Unsupported, undocumented.
 
+parser.add_option('--enable-trace-maps',
+    action='store_true',
+    dest='trace_maps',
+    help='Enable the --trace-maps flag in v8')
```

Comment:
I'm +1 to documenting with a warning. Quite a few of `configure`'s switches have wide reaching results...